src/scripts/app.ts

```diff
@@ -2360,9 +2389,13 @@ export class ComfyApp {
       this.#invokeExtensions('loadedGraphNode', node)
     }
 
+    // TODO: Properly handle if both nodes and models are missing (sequential dialogs?)
```

Comment:
In the main PR description I wrote:
`Does not yet handle the case of both models and nodes are missing properly`
`handling this well might need coordinating with Comfy Manager and/or merging Manager in? Since I imagine manager overrides this whole dialog and all anyway.`
I'm hesitant to merge the dialogs right away due to the increased potential conflict, unless you know confidently that the existing dialog can be mucked up without it being an issue